src/OmniSharp.MSBuild/ProjectFile/PropertyConverter.cs

```diff
@@ -165,10 +165,9 @@ public static OutputKind ToOutputKind(string propertyValue)
             }
         }
 
-        public static NuGetVersion ToNuGetVersion(string propertyValue)
+        public static VersionRange ToVersionRange(string propertyValue)
         {
-            NuGetVersion version;
-            if (NuGetVersion.TryParse(propertyValue.Trim(), out version))
+            if (VersionRange.TryParse(propertyValue.Trim(), out var version))
```

Comment:
maybe I'm weird but doesn't it strike you as a bit inconsistent that in the same namespace they'd have two different naming styles - `NuGetVersion` and `VersionRange` (used for, well, `NuGetVersions`)?